src/Contract/Operation.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace loophp\collection\Contract;
+
+use Closure;
```

Comment:
Yes, I'm mitigated.

According to me, a closure is an anonymous function, and it's also a callable.
On the other hand, a callable is not necessarily a Closure. It can also be: `['Class', 'Method']`.

This is why I'm doing a distinction.